ReleaseChecklist.md

```diff
@@ -1,24 +1,52 @@
-Checklist for making a release:
+## Checklist for making a release:
 
+### Requirements
+ - [ ] AppVeyor account
+ - [ ] Lauchpad (Ubuntu One) account
+ - [ ] gnupg key (has to be version 1, gpg2 won't work) for `your-name@ethereum.org` created and uploaded
```

Comment:
@chriseth I'm not sure that this works in the end, because ``release_ppa.sh`` would need to be changed properly. And I can't tell right now, if changing the email and the key is enough. We could make these arguments to the script, though.